endpoints-support.md

```diff
@@ -69,18 +69,23 @@
-|<sub>/rest/logical-enclosures/{id}</sub>                                                 | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}</sub>                                                 | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}</sub>                                                 | PATCH    | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}</sub>                                                 | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}/configuration</sub>                                   | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}/script</sub>                                          | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}/script</sub>                                          | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}/support-dumps</sub>                                   | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
-|<sub>/rest/logical-enclosures/{id}/updateFromGroup</sub>                                 | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:  | :white_check_mark:  | :white_check_mark:  |
+|   **Interconnects**                                                                                                                           |
+|<sub>/rest/interconnects/{id}</sub>                                                    | PATCH    | :white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  |
+|<sub>/rest/interconnects/{id}/configuration</sub>                                      | PUT    | :white_check_mark:  | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  |
```

Comment:
This endpoint supported from API 500 onwards.please mark it correctly here

Reply:
@soodpr Done